util/version/version_test.go

```diff
@@ -268,7 +268,38 @@ func TestCompareWithBuildIdentifiers(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			g := NewWithT(t)
-			g.Expect(CompareWithBuildIdentifiers(tt.a, tt.b)).To(Equal(tt.expected))
+			g.Expect(compareWithBuildIdentifiers(tt.a, tt.b)).To(Equal(tt.expected))
+		})
+	}
+}
+
+func TestCompare(t *testing.T) {
+	tests := []struct {
+		name     string
+		aVersion semver.Version
+		bVersion semver.Version
+		options  []CompareOption
+		want     int
+	}{
+		{
```

Comment:
Can we have a test that compares two equal version but with different build tags, thus proving that default compare ignores build tags?

Reply:
@ykakarap am I wrong or we still missing this test

Reply:
@fabriziopandini  L290-295 now has the correction versions for the test.